tests/test_vcf_writer.py

```diff
@@ -64,3 +64,33 @@ def test_write_filetypes(self, tmp_path, output_suffix, input_file, write_header
         with VcfReader(input_file) as reader, VcfWriter(output_path) as writer:
             writer.write(headers=reader.headers, variants=reader)
         assert open_file(output_path).read(9999999).encode().strip() == Path(expected_output_file).read_bytes().strip()
+
+    @pytest.mark.parametrize("input_file, expected_output_file", (
+            ('tests/samples/sample1.vcf', 'tests/samples/sample1.info_rank.vcf'),
+    ))
+    def test_write_non_sorted_headers_at_the_end(self, input_file, expected_output_file):
```

Comment:
use tmp_path for having a temporary folder

Reply:
look at line 72